source/store/operations_and_invariants/bool_invariants.py

```diff
@@ -342,6 +316,32 @@ def print(graph, precision):
         return Utils.print_boolean(SomeEigenIntegerDQ.calculate(graph), precision)
 
 
+class SomeEigenIntegerS(InvariantBool):
```

Comment:
Several of the eigenValues have been changed to eigen, but not this one. Should it remain unchanged?

Reply:
I also notice that there are changes in PR #418  in the eigenValue, I recommend keeping an eye out so as not to merge changes that return the names to their original states

Reply:
But I changed only the display name, not class name.